rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2485,6 +2490,22 @@ def test_upgrade_db_31_to_32(user_data_dir):  # pylint: disable=unused-argument
             'SELECT COUNT(*) FROM sqlite_master WHERE type="table" AND name=?', (name,),
         ).fetchone()[0] == 0
 
+    manual_balance_after = cursor.execute(
+        'SELECT asset, label, amount, location, category FROM '
+        'manually_tracked_balances;',
+    ).fetchall()
+
+    manual_balance_right = [
```

Comment:
nitpick for the naming here. For such cases in the code we use the term "expected"

```suggestion
    manual_balance_expected = [
```